db/schema.rb

```diff
@@ -91,6 +96,15 @@
     t.index ["vendor_id"], name: "index_vendor_ownerships_on_vendor_id"
   end
 
+  create_table "vendor_tags", force: :cascade do |t|
```

Comment:
I think this still shouldn't be here. Might need to run the reset David mentioned again.